src/components/CytoscapeGraph/graphs/TrafficRenderer.ts

```diff
@@ -84,6 +100,23 @@ class TrafficPointGenerator {
   setSpeed(speed: number) {
     this.speed = speed;
   }
+
+  setErrorRate(errorRate: number) {
+    this.errorRate = errorRate;
+  }
+
+  private nextPoint(): TrafficPoint {
+    console.log('errorRate', this.errorRate);
+    const isErrorPoint = Math.random() <= this.errorRate;
+    console.log('isErrorPoint', isErrorPoint);
+    console.log('speed', this.speed);
```

Comment:
people have been complaining lately about the amount of logs. So either remove these or change to debug logs

Reply:
forgot to remove, thanks